docs/cudf/source/internals.md

```diff
@@ -0,0 +1,95 @@
+cuDF internals
+==============
+
+ColumnAccessor
+--------------
+
+The underlying data structure of cuDF `Series`, `DataFrame` and `Index` objects
+is a `ColumnAccessor`, which can be accessed via their `._data` attribute:
```

Comment:
Should indicate that `ColumnAccessor` is underneath `Frame` which those classes inherit from.